README.md

```diff
@@ -7,13 +7,17 @@ In this project, we've curated a collection of UI elements, layouts, and interac
 ## Packages
 
 In order to mantain this project with the most updated version of LTS's packages:
+
 Run `npm outdated` to see a table of packages with the current version, wanted version, and latest version.
+
 To update a specific package, you can use `npm update package_name` This will update the package to the 'wanted' version, which is the maximum version that satisfies the versioning range specified in package.json.
 
```

Comment:
I'd add suggestion to use [npx ](https://www.npmjs.com/package/npm-check-updates#installation) which is even better. I'd add it below, as the npm outdated is the 'official' one.

I'd add a strong suggestion to use npm-check-updates (ncu) via npx which is fantastic.